lib/basevm.py

```diff
@@ -63,6 +63,12 @@ def all_vdis_on_host(self, host):
                 return False
         return True
 
+    def all_vdis_on_sr(self, sr):
```

Comment:
Do we still need the all_vdis_on_host method? If yes, should we make it return False for shared SRs?